client-coroutines/src/main/kotlin/io/provenance/client/coroutines/gas/estimators/Floating.kt

```diff
@@ -6,15 +6,15 @@ import io.provenance.client.coroutines.gas.prices.GasPrices
 import io.provenance.client.coroutines.gasEstimator
 
 internal fun floatingGasPriceGasEstimator(delegate: GasEstimator, floatingGasPrice: GasPrices) = gasEstimator { tx, adjustment ->
-    val (price, denom) = floatingGasPrice().let { it.amount.toDouble() to it.denom }
+    val (price, denom) = floatingGasPrice().let { it.amount.toBigDecimal() to it.denom }
```

Comment:
It's worth fixing in the non-coroutines client as well: https://github.com/provenance-io/pb-grpc-client-kotlin/blob/298ac90cef282c6e2609a3de82b795079ceb4036/client/src/main/kotlin/io/provenance/client/gas/estimators/Floating.kt#L9

Reply:
thanks @benarena - didn't even think to check for dupes